packages/store/addon/-private/system/promise-proxies.js

```diff
@@ -49,7 +49,7 @@ export const PromiseArray = ArrayProxy.extend(PromiseProxyMixin, {
   be updated when the promise resolves.
 
   For more information see the [Ember.PromiseProxyMixin
-  documentation](/api/classes/Ember.PromiseProxyMixin.html).
+  documentation](/api/classes/PromiseProxyMixin.html).
```

Comment:
```suggestion
  documentation](/ember/release/classes/PromiseProxyMixin.html).
```

I haven't had a chance to test this locally yet but it sounds like from the comments that this will work to map to this link: https://api.emberjs.com/ember/release/classes/PromiseProxyMixin